shared/src/main/scala/mlscript/NuTypeDefs.scala

```diff
@@ -389,6 +389,11 @@ class NuTypeDefs extends ConstraintSolver { self: Typer =>
       this
   }
   
+  // Store referred members. indirect ref: ref by using `this`
```

Comment:
Comment is impossible to understand. Please clearly state what this stores and the meaning of the two `Bool` uses.

Reply:
Updated the comment here.